src/App.js

```diff
@@ -9,6 +10,7 @@ const App = () => {
         <Routes>
           <Route path="/" element={<SearchScreen />} />
           <Route path="/search" element={<SearchResult />} />
+          <Route path="/invoice" element={<InvoiceScreen />} />
```

Comment:
Should We use the same name as in schema; `Quote`? verify with schema

Reply:
The Quote is selectJourney Modal and this invoice is final confirmation screen.
Though we specify route for invoice the invoice is not integrated in workflow till now.